main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     bike
+//  action 1: tires pop
+bike.popTire();
+//  action 2: stops
+bike.stop();
+//  action 3: accelerates
+bike.accelerate();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     elevator
+//  action 1: close doors
+elevator.closeDoors();
+//  action 2: calls help
+elevator.connectToEmergencyServices();
+//  action 3: goes to lobby
+elevator.goToLobby()
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     guitar
+//  action 1: string breaks
+guitar.breakString();
+//  action 2: increase pitch of string
+guitar.increaseStringPitch();
+//  action 3: switch pickout wiring
+guitar.outputPickup();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     tv
+//  action 1: display different channel
+tv.newChannel();
```

Comment:
your action is `display different channel`
your pseudo code says `newChannel()` which is not the same thing.
your pseudo code should be
```
tv.displayDifferentChannel();
```
This concept of converting plain english expressions into pseudo-code is an important concept.
It's important because it helps make your code read almost like Plain English.
This makes your code easier to understand and self-documenting.
- [ ] revise